gradle/libs.versions.toml

```diff
@@ -14,6 +14,9 @@ dokka = '1.7.20'
 androidxLifecycle = "2.6.0-alpha01"
 coil = '2.2.0'
 
+kotlinCompilerExtension = "1.4.7"
+composeBom = "2023.05.01"
```

Comment:
A newer version of androidx.compose:compose-bom than 2023.05.01 is available: 2023.08.00

Reply:
I update kotlinCompilerExtension and composeBom to the compatible with Kotlin 1.8.22 versions